analyzer/javacg-opal/src/test/java/eu/fasten/analyzer/javacgopal/data/JSONUtilsTest.java

```diff
@@ -0,0 +1,157 @@
+package eu.fasten.analyzer.javacgopal.data;
```

Comment:
I'm curious why the test class is not in the same package as the tested class (`eu.fasten.core.data`)?

Reply:
I would prefer to do some heavy testings by comparing a large number of maven packages JSON strings, for that I selected 100 random maven coordinates. Creating a call graph out of the maven coordinates feature is only available in the opal plugin. If I want to use this feature from the core I would have to declare a dependency to the opal plugin and it would be a cyclic dependency.

Reply:
Go it, thanks for the clarification.